src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java

```diff
@@ -14,5 +15,6 @@
    * @return ActionPlanDTO representation of the created action plan
    * @throws RestClientException for failed connection to action service
    */
-  ActionPlanDTO createActionPlan(String name, String description) throws RestClientException;
+  ActionPlanDTO createActionPlan(String name, String description, HashMap<String, String> selectors)
```

Comment:
It's nicer for the API to define a Map as a parameter rather than the exact type of map. Allows the consumer of the API to pass any type of Map

Reply:
:+1: I'll add another card for that as it'd affect a few other repos

Reply:
you should be able to use the base type Map without affecting any other repos - the JSON serialisation will be identical

Reply:
Please update the method comments (Java docs ) or remove them ( i prefer we do not use code comments , the code should be self explanatory ).